QgisModelBaker/libili2db/iliexecutable.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+    begin                :    09/09/23
+    git sha              :    :%H$
+    copyright            :    (C) 2020 by Yesid Polania
+    email                :    yesidpol.3@gmail.com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+import functools
+import locale
+import re
+from qgis.PyQt.QtCore import QObject, pyqtSignal, QProcess, QEventLoop
+from abc import ABCMeta, abstractmethod
+
+from QgisModelBaker.libili2db.ili2dbutils import get_java_path, get_ili2db_bin, JavaNotFoundError
+from QgisModelBaker.libqgsprojectgen.db_factory.db_simple_factory import DbSimpleFactory
+
+
+class AbstractQObjectMeta(ABCMeta, type(QObject)):
```

Comment:
Can you add a docstring that explains the reasons for this this class?

Reply:
Done! Besides, I moved this class to qt_utils file because it may be used by other classes

Reply:
Good move